system-tests/projects/invalid-root-level-config/invalid-component-experimentalSessionAndOrigin-config.js

```diff
@@ -1,8 +0,0 @@
-module.exports = {
-  component: {
-    devServer: {
-      framework: 'create-react-app',
-      bundler: 'webpack',
-    },
-  },
-}
```

Comment:
Does this not need a replacement with another framework?

Reply:
it gets replaced by `react` and `webpack` which we already support. but this file in general looks to be left over from when sessionAndOrigin went Ga so its dead code